### ISSUE TITLE
Inline scripts trigger no-multiple-empty-lines incorrectly

### ISSUE THREAD

**Issue by avrelaun** [BenoitZugmeyer/eslint-plugin-html#62]
I updated recently eslint-plugin-html to 3.0.0, and now I have false flags on my .vue components.

"warning  Too many blank lines at the beginning of file. Max of 0 allowed  no-multiple-empty-lines"

```
<template>
<div>
</div>
</template>
<script> <- Too many blank lines here
import stuff
export default {
};
</script>
```

**Comment by BenoitZugmeyer** [comment]
This is expected: as of 3.0.0, the plugin will lint script tags independently, and you have a new-line at the beginning of the script. See the [migration guide](https://github.com/BenoitZugmeyer/eslint-plugin-html/blob/master/MIGRATION_TO_V3.md) to understand the impacts of this move.

As of ESLint 4.1.0, you could use [an "overrides" option](http://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns) to customize the configuration for vue files:
```yaml
"overrides": [
  {
    "files": [ "*.vue" ],
    "rules": {
        "no-multiple-empty-lines": ["error", { "maxBOF": 1 }]
    }
}
```
